is/esb/newmount.sh

```diff
@@ -29,19 +28,20 @@ sudo apt-get -y install nfs-common
 sudo apt-get -y install nfs-kernel-server
 
 #create the drive that is being mapped to if doesn't exist
-if [ ! -d $4 ]; then
-  echo "missing directory, creating $4"
-  sudo mkdir -p $4
-  sudo chmod 777 $4
+if [ ! -d $3 ]; then
+  echo "missing directory, creating $3"
+  sudo mkdir -p $3
+  sudo chmod 777 $3
 fi
 
-# Map storage account to drive if 4 parameters exist.
-if [ ! -z $4 ]; then
+# Map storage account to drive if required parameters exist.
+if [ ! -z $3 ]; then
   #added in code to check if the map data command is already in the fstab file
-  MAPVALUE="//mount -t nfs $1.file.core.windows.net/$3 $4 -o vers=4,minorversion=1,sec=sys,nconnect=4"
+  MAPVALUE="$1.file.core.windows.net/$2 $3 nfs  vers=4,minorversion=1,sec=sys"
```

Comment:
Your mapping syntax seems to still be for cifs and not nfs

Reply:
Thanks, John. i have made the required changes